guides/plugins/plugins/administration/add-rule-assignment-configuration.md

```diff
@@ -0,0 +1,290 @@
+# Add rule assignment configuration
+
+## Overview
+
+You want to create an own card in the rule assignment, where you can add or delete assignments? This guide gets you covered on this topic. Based on an example of the configuration of the `Dynamic Access` plugin, you will see how to write your configuration.
+
+## Prerequisites
+
+This guide **does not** explain how to create a new plugin for Shopware 6.
+Head over to our Plugin base guide to learn how to create a plugin at first:
+
+{% page-ref page="../plugin-base-guide.md" %}
+
+## Creating the index.js file
+
+The first step is creating a new directory like `<plugin root>/src/Resources/app/administration/src/module/sw-settings-rule/extension/sw-settings-rule-detail-assignments`, so you store the extending component file in there.
```

Comment:
Please rewrite the sentence to "... a new directory like so <PATH>". The rest is obsolete.